_posts/2020-10-06-solidity-dynamic-array-cleanup-bug.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: post
+published: true
+title:  "Solidity Dynamic Array Cleanup Bug"
+date:   2020-10-06
+author: Solidity Team
+category: Security Alerts
+---
+
+On September 17, 2020, a bug in the Solidity code generator was found. The bug is fixed with version 0.7.3
+released on October 6, 2020. The bug is present in all prior versions of Solidity.
+
+We assigned the bug, a severity level of *medium*.
+
+## Dynamic Array Cleanup Bug
```

Comment:
```suggestion
## Technical Details of the Bug
```

Reply:
Otherwise the article of the article is "Dynamic Array Cleanup Bug" and the sub-heading is the same.